scipy/optimize/_linprog.py

```diff
@@ -557,10 +568,21 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     # need modified bounds here to translate variables appropriately
     disp = solver_options.get('disp', False)
 
-    x, fun, slack, con, status, message = _postprocess(x, postsolve_args,
-                                                       complete, status,
-                                                       message, tol,
-                                                       iteration, disp)
+    # print("linprog() bounds: input to postsolve ", postsolve_args[0][5])
+
+    # x, fun, slack, con, status, message = _postprocess(x, postsolve_args,
+    #                                                    complete, status,
+    #                                                    message, tol,
+    #                                                    iteration, disp)
+
+    x, fun, slack, con = _postsolve(x, postsolve_args, complete, tol)
+
+    status, message = _check_result(x, fun, status, slack, con, lp_o.bounds, tol, message)
+    # not bounds output from _postsolve anymore, because the reconstruction
+    # there is removed and the return argument as well
```

Comment:
Just checking understanding - there was no reason to reconstruct the bounds from `undo` because the unmodified bounds are just available in `lp_o`, and there was no other reason to perform the reconstruction?

Reply:
That is right.